sample-notes.md

```diff
@@ -0,0 +1,10 @@
+# 2019-05-16 Example
+
+## Notes
+
+- e.g. this is an incorrect line.
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/patcon/dangerfile-test/blob/4fda633c79d810bac673fe0baff393606ed56acb/sample-notes.md#L5"></span>Use ‘e.g.,’ instead of ‘e.g.’.</td>
    </tr>
  </tbody>
</table>